devise_google_authenticator.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |s|
     # removed the following to try and get past this bundle update not finding compatible versions for gem issue
     # 'actionmailer' => '>= 3.0',
     # 'actionmailer' => '~> 3.2',# '>= 3.2.12',
-    'devise'  => '4.6.2',
+    'devise'  => '>= 4.6.2',
     'rotp'    => '>= 1.6',
     'rqrcode' => '>= 0.10.1'
```

Comment:
Shouldn't we use `~>` instead of `>=` for all dependencies?

Reply:
Makes sense